content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
 | Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
 | Situation       |     2/0 |     4/0 | SIT                  |
 
+Question Type Minimums
+: For a given Question Type, the minimum number of questions must be asked in Questions Numbered 1-20. This excludes any questions appended with "A" or "B."
+
+Question Type Maximums
+: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any questions appended with "A" or "B."
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
```

Comment:
I also re-wrote this to be a little more robust. But I think it maintains functional parity.

Reply:
I don't think so. "Negative" answers are never acceptable as answers to a question, due to a different rule outside the MA section. When this rule is phrased this way, it could confuse people into thinking the question IS valid if it has two negative answers, which isn't true. I would keep the original phrasing here.

Reply:
So then that's a failing of the current rulebook. It currently says "A negative and positive answer to a question is not a valid MA." But it's really already covered by the "if the question is not answered."   So there's slightly different wording to apply to the same thing. 

I was just rewording the first to make it clear it's not limited to a 2 answer question, where one answer is positive and one is negative.

Reply:
You're both right. I think the change here is OK and func equiv in and of itself. I think in v-next we can start iterating language clarity improvements.

Reply:
The rewording is more confusing than the original wording, IMO. Although the rule itself is stupid, I think the original rulebook states it fine by saying elsewhere that you can't have negative answers, and then making it especially clear in the MA section where it's most likely to have issues.

Reply:
Should the language under MAs be an example? 

@gryphonshafer I'm a bit unclear on your thoughts. You said my change is Ok and FE. But then you said language clarity improvements should come in v-next....which is pretty much what my change is.

Reply:
I think both are true. I think your changes are OK/good for v-1, and we have the opportunity for more improvements in v-next. That said, Jeremy thinks the original is better than your proposed changes. He might be right, but I'm inclined to think your changes are better.

Reply:
I'd love further thoughts from @jswingle and initial thoughts from @ztinker before making a decision here and resolving the thread.

Reply:
I might reword it something like this. 

If one of the listed answers to a the multiple answers questions is a "negative" answer, this is deemed invalid as a Multiple Answer question.